### PR TITLE
Curves

### PR DESCRIPTION
**Pull request by Mauvehill**
Request to add new scenarios to the Test framework.

### CODE REVIEW

File: src/steps/curves-club.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    browser.url('https://www.curvesclublingerie.com/');
+    // browser.pause('5000');
+})
+
+When('I click Styles Solutions tab', function() {
+    console.log('click step');
+    const styleSolutions = browser.$('#et-navigation li:nth-child(3) a');
+    styleSolutions.waitForDisplayed();
+    styleSolutions.click();
+    // browser.debug();
```
Comment (jimode) [inline src/steps/curves-club.js]: It seems like you left some comments in your steps. Do you want to remove those?

File: src/steps/curves-club.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Given, When, Then } from 'cucumber';
+
+Given('I visit the Curves club lingerie website', function() { 
+    browser.url('https://www.curvesclublingerie.com/');
+    // browser.pause('5000');
```
Comment (jimode) [inline src/steps/curves-club.js]: It seems like you left some comments in your steps. Do you want to remove those?

**Review by jimode — CHANGES_REQUESTED** [review]

File: src/steps/curves-club.js (hunk 1 above)
Comment (Mauvehill) [reply src/steps/curves-club.js]: comments removed.

File: src/steps/curves-club.js (hunk 2 above)
Comment (Mauvehill) [reply src/steps/curves-club.js]: comments removed.

**Review by jimode — APPROVED** [review]

**Pull request merged**